xarray/tests/test_dask.py

```diff
@@ -292,6 +293,22 @@ def test_persist(self):
         self.assertLazyAndAllClose(u + 1, v)
         self.assertLazyAndAllClose(u + 1, v2)
 
+    @requires_pint_0_15
```

Comment:
could you add a comment stating that this is necessary because we need `__dask_tokenize__`? That way deciding when to remove this will be easier.

Reply:
Yes, in `__init__.py`, `test_dask.py`, or both?

Reply:
in `test_dask.py` should be enough